File: coinflip.cs
using System;

public class Coinflipping
{
    public static void Main(string[] args)
    {
        int times;
        int head = 0;
        int tail = 0;
        int dif = 0;
        int ran;
        Console.WriteLine ("How many times should I flip the coin? ");
        times = Convert.ToInt32(Console.ReadLine());
        Random random = new Random();
        for(int i = 0; i < times;i ++){
            ran = random.Next(2);
            if(ran == 0){
                head++;
            }
            else{
                tail++;
            }
        }
        Console.WriteLine("Number of heads: {0}", head);
        Console.WriteLine("Number of tails: {0}", tail);
        if(head > tail){
            Console.WriteLine("head wins");
            dif = head - tail;
        }
        else if(tail > head){
            Console.WriteLine("tail wins");
            dif = tail - head;
        }
        else{
            Console.WriteLine("draw");
        }
        Console.WriteLine("The difference: {0}", dif);
    }
}

File: coinflip.js
let ran = 0;
let time = prompt("How many times should I flip the coin");
let head = 0;
let tail = 0;
let dif = 0;
//flipping coin
for(let i = 0; i < time; i++){
    ran = Math.round(Math.random()); // random num (0 or 1)
    if( ran == 1){
    head ++;
    }
    else{
        tail ++;
    }
}
console.log("Number fo heads: " + head);
console.log("Number of tails: " + tail);
if(head > tail){
    console.log("head wins");
    dif = head - tail;
}
else if(tail > head){
    console.log("tail wins")
    dif = tail - head;
}
else{
    console.log("draw");
}
console.log("The difference is: " + dif)